src/pykiso/interfaces/dt_auxiliary.py

```diff
@@ -96,6 +96,7 @@ def run_command(
         cmd_data: Any = None,
         blocking: bool = True,
         timeout_in_s: int = 5,
+        timeout_result: Any = None,
```

Comment:
Who should set this value? run_command can potentially be called by a tester script right?

Reply:
Absolutely, this just allows returning a different value than None on a timeout of the command, which removes any need to override run_command in case a more meaningful return value is wanted